components/script/script_thread.rs

```diff
@@ -2016,6 +2016,7 @@ impl ScriptThread {
             WebGPUMsg::FreeCommandBuffer(id) => self.gpu_id_hub.lock().kill_command_buffer_id(id),
             WebGPUMsg::FreeSampler(id) => self.gpu_id_hub.lock().kill_sampler_id(id),
             WebGPUMsg::FreeShaderModule(id) => self.gpu_id_hub.lock().kill_shader_module_id(id),
+            WebGPUMsg::Exit => *self.webgpu_port.borrow_mut() = None,
```

Comment:
How about, instead of an exit message, we do something like:

```rust
if msg.is_err() {
    *self.webgpu_port.borrow_mut() = None;
    return true;
}
FromWebGPUServer(msg.unwrap())
```

Just in case that also works, otherwise it's not a big deal, however I am trying to not add more explicit exit messages in the light of https://github.com/servo/servo/issues/26502

Reply:
Ah it's awaiting merge already, nevermind, another time...